src/rdkafka_broker.c

```diff
@@ -3658,6 +3709,13 @@ rd_kafka_fetch_reply_handle (rd_kafka_broker_t *rkb,
 					  Throttle_Time);
 	}
 
+        if (rd_kafka_buf_ApiVersion(request) >= 7) {
+                int16_t ErrorCode;
```

Comment:
my implicit assumption was we'd never get an error here since it's only related to KIP-227, but that's a really bad assumption (though likely true at this point based on every thing I know). well pointed out... In the kip it says:

> When the top-level error code is set, the caller should assume that all the partitions in the fetch received the given error.

so i'm going to do that - blindly replace the per partition error variable with this if set. then let it fall through to the default handler which creates an error op (non-retryable error).